submissions/LuckyDnepr/js-dom/main-script.js

```diff
@@ -0,0 +1,63 @@
+"use strict";
+
+import { readJSON } from "./readJSON.js";
+
+import { loadContentByID } from "./aside-menu-action.js";
+
+const dataBaseUrl = "./data-films.json";
+
+const asideMenu = document.querySelector("#aside_menu"),
+  mainContainer = document.querySelector("#content");
+
+makeAsideMenu(dataBaseUrl, asideMenu)
+  .then((asideMenu) => addAsideMenuListener(asideMenu))
+  .catch((error) => showSorryMessage(mainContainer, error));
+
+async function makeAsideMenu(source, container) {
+  const data = await readJSON(source);
+  const asideMenuList = document.createElement("ul");
+  asideMenuList.classList.add("aside_menu_list");
+  data.forEach((film) => {
+    asideMenuList.appendChild(makeAsideMenuItem(film.ID, film.title));
+  });
+  container.appendChild(asideMenuList);
+  return asideMenu;
```

Comment:
This looks a little confusing. The purpose of this function is to render an AsideMenu. When you name function `makeAsideMenu` it usually means that it should return some kind of element, and this function does do that, but you're not using this element in any way, because what it also does is render the element on line `23` right away. Either rename the function to something like `renderAsideMenu` and remove the return or keep the return, remove rendering here and render somewhere outside this function using the returned value.